src/Module/Api/Mastodon/Accounts/Statuses.php

```diff
@@ -113,11 +113,13 @@ protected function rawContent(array $request = [])
 			$items = Post::selectForUser($uid, ['uri-id'], $condition, $params);
 		}
 
+		$display_quotes = self::appSupportsQuotes();
+
 		$statuses = [];
 		while ($item = Post::fetch($items)) {
 			self::setBoundaries($item['uri-id']);
 			try {
-				$statuses[] = DI::mstdnStatus()->createFromUriId($item['uri-id'], $uid);
+				$statuses[] = DI::mstdnStatus()->createFromUriId($item['uri-id'], $uid, true, true, $display_quotes);
```

Comment:
Since the last parameter seems to be mandatory now, please alter the order of its parameters so that the optional parameters are at the end.

Calls should look like this when you're done.
```suggestion
				$statuses[] = DI::mstdnStatus()->createFromUriId($item['uri-id'], $uid, $display_quotes);
```

Reply:
Done